apiary.apib

```diff
@@ -5883,18 +5880,16 @@ For a more detailed guide on the file upload, see the corresponding part of the
     + sizeBytes (optional, number) - File size in bytes
     + contentType (optional) - File [MIME Type](https://en.wikipedia.org/wiki/Media_type)
         Default: `application/octet-stream`
-    + isPublic (optional, boolean) - When true, the file URL will be permanent and publicly accessible.
-        Default: false
```

Comment:
Slyšel jsem, že se toho chceme zbavit, tak by mohlo být dobrý nápad to odtud odstranit.

Reply:
Podporované už je to pouze v AWS. Asi bych spíš zatím označil jako DEPRECATED a smazal až to bude úplně pryč.

Reply:
Podle mě je to OK to takto odstranit z té dokumentace, už nechceme aby to někdo používal.